code/models.py

```diff
@@ -0,0 +1,52 @@
+'''
```

Comment:
So are we going to use this? If not then maybe don't check it in. I can merge in my file and make it talk nice to `data.py`.

Generally, I think now is a good time to agree on an interface for models. I was thinking of something along the lines of a stripped down sklearn type of thing with a simple `fit` and `predict` and maybe `predict_proba`. @StefanGStark suggested something from the Stanford NLU course. I strongly believe that we should have an interface but don't have a strong opinion as to which form it should take...as long as it doesn't slow anyone down too much.